File: Framework-Samples/NHibernate/CSharp/Sample.Data/ManagerObjects/ProfileManager.cs
using System;
using System.Collections.Generic;
using System.Text;

using NHibernate;
using Sample.Data.Generated.BusinessObjects;
using Sample.Data.Generated.Base;

namespace Sample.Data.Generated.ManagerObjects
{
    public partial interface IProfileManager : IManagerBase<Profile, int>
    {
	}
	
	partial class ProfileManager : ManagerBase<Profile, int>, IProfileManager
    {
	}
}

File: Framework-Samples/NHibernate/VisualBasic/Sample.Data/ManagerObjects/ProfileManager.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Imports NHibernate
Imports Sample.Data.Generated.BusinessObjects
Imports Sample.Data.Generated.Base

Namespace Sample.Data.Generated.ManagerObjects
	'Public Partial Interface IProfileManager
	'	Inherits IManagerBase(Of Profile, System.Int32)
	'End Interface

	Partial Class ProfileManager
		Inherits ManagerBase(Of Profile, System.Int32)
		Implements IProfileManager
	End Class
End Namespace
